README.md

```diff
@@ -2,45 +2,43 @@
 
 [![Build Status](https://travis-ci.org/mapbox/patrol-rules-github.svg?branch=master)](https://travis-ci.org/mapbox/patrol-rules-github)
 
-A set of rules implemented using [lambda-cfn](https://github.com/mapbox/lambda-cfn) and designed to run on a [Patrol](https://github.com/mapbox/patrol) stack.  The rules in this repository all aim to monitor an organizations Github repositories and users for best practices, security and compliance. Read more about the Patrol architecture on the [Patrol project](https://github.com/mapbox/patrol).
+A set of functions implemented using [lambda-cfn](https://github.com/mapbox/lambda-cfn) to monitor an organizations Github repositories and users for best practices, security and compliance. Part of the Mapbox [Patrol](https://github.com/mapbox/patrol) security framework.
 
-### Usage
+### Deploying
 
-Follow the steps on the [Patrol](https://github.com/mapbox/patrol) readme to set up your own Patrol stack on AWS which makes use of the patrol-rules-github rules.  Follow instructions on Patrol on how to enable or disable particular rules and how to deploy on your own AWS account.
+Please see the [lambda-cfn README](https://github.com/mapbox/lambda-cfn)
```

Comment:
We shouldn't only link to the lambda-cfn README here, we should be more explicit. While the new lambda-cfn documentation is very detailed it may still be somewhat abstract to some people, especially those new to lambda-cfn and Patrol. This will also save us a ton of time when new people want to write new Patrol rules or troubleshoot existing ones.

We should add a section and write docs on how to create a new Patrol rule in this project. It would include docs on how to run `lambda-cfn init` and in which directory people should run this command (e.g. the parent directory, not a rules sub directory). We would then describe how to run `lambda-cfn create` to create and deploy the new rule stack. 

We should create another section on how to update existing rules for when users make a change to the code or the configuration. We would describe how a user would `cd` into a Patrol rule directory then run `lambda-cfn update` to update an existing rule.

Some of these docs overlap a lot with /patrol-rules-aws - we can use the /patrol repo to keep all of this documentation in one place if that's easier.

Note: these docs wouldn't include any sensitive or private details on how to deploy Patrol at Mapbox. We would keep these in our own private repo.